cdci_data_analysis/analysis/hash.py

```diff
@@ -34,3 +34,12 @@ def format_hash(x): return hashlib.md5(
 
     # this takes care of various strange objects which can not be properly represented
     return format_hash(json.dumps(o))
+
+
+def make_hash_file(file_path):
+
+    # TODO to adapt in case of big files and reading in chunks will be necessary
```

Comment:
I'd just run `md5sum` with subprocess, it's likely to be faster and memory-preserving even for moderately big files.

Reply:
it now opens in a subprocess